### ISSUE TITLE
Users getting fake error on posting announcements

### ISSUE THREAD

**Issue by micheal-cooper** [ywc/ywc#21]
User tries to upload a post to All-OIST Announcements and gets "Your post could not be saved" and tries again and again, but in reality, the system is saving the new posts and sending out emails for each attempt.

We fixed this before, I believe, by limiting the number of nodes returned by the View on the Drupal side. I just limited the number of nodes to 100 for announcements. Hopefully that will do it.

**Comment by micheal-cooper** [comment]
Yes, it looks like we fixed Marketplace by my limiting the nodes returned in the xml feed, so I have limited announcements similarly, and it looks like it worked. Just did a successful test.

**Comment by topherwhite** [comment]
Thanks for the fix but this is not a great solution, cause we're limiting the data available to users. Great for now, but I need to implement multi layered caching in ywc and also make it async. Thanks!

**Comment by micheal-cooper** [comment]
It's back again. I am getting complaints and requests to unsubscribe from TIDA because people are receiving duplicate email notifications, which is because the poster gets the attached error and tries to post again, not knowing that the post WAS saved and the email notifications have gone out.

Just to quickly respond to this big issue, I will further reduce the results given in the feed.
![screen shot 2014-02-27 at 11 23 48](https://f.cloud.github.com/assets/3095809/2278525/d482d8c2-9f56-11e3-9583-83c54c589d72.jpg)

**Comment by micheal-cooper** [comment]
Topher, please take a look.

**Comment by micheal-cooper** [comment]
Topher, even after I have reduced the xml feed for announcements to only the last 25 items, people are still getting this fake error when posting to Announcements. As a result, they try again, and again, and each time a new email is being sent, so one faculty member is very vocally asking to be removed from the subscription list, and other people, including a VP and Neil, are asking that something be done soon.

I will turn off the email notifications until you can look at this. Please turn them back on when you think it is safe to do so. Thank you.

**Comment by micheal-cooper** [comment]
I think our mailing list has reached a new watershed, so posters are timing out, getting error messages, and reposting again and again. I increased the timeout in httpd.conf from 60 to 90, but that did not work. I am looking for records of our fix to this. I know that you increased the timeout limit somewhere, but I cannot find it in ywc.properties or httpd.conf. I am still searching for where I can change timeout. Drop me a hint if you can.
